### PR TITLE
fix: Apply default style changes to the code

### PR DESCRIPTION
**Pull request by wtrocki**
### Motivation

Run lint and formatter (it looks like most of the code were not processed by formatter and linter)

### CODE REVIEW

File: modules/core/metrics/impl/DeviceMetrics.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class DeviceMetrics: Metrics {
         return [
             "platform": "ios",
             "platformVersion": UIDevice.current.systemVersion,
-            "device": UIDevice.current.model
+            "device": UIDevice.current.model,
```
Comment (danielpassos) [inline modules/core/metrics/impl/DeviceMetrics.swift]: Should we have `,` in the last line?

File: modules/core/metrics/impl/AppMetrics.swift [hunk 2]
```diff
@@ -20,7 +20,7 @@ public class AppMetrics: Metrics {
         return [
             "appId": self.appData.bundleId,
             "appVersion": self.appData.appVersion,
-            "sdkVersion": self.appData.sdkVersion
+            "sdkVersion": self.appData.sdkVersion,
```
Comment (danielpassos) [inline modules/core/metrics/impl/AppMetrics.swift]: Should we have `,` in the last line?

Comment (wtrocki) [reply modules/core/metrics/impl/AppMetrics.swift]: This is what lint does. Going to alter lint config to not insert that.

File: modules/core/metrics/AgsMetrics.swift [hunk 3]
```diff
@@ -63,7 +63,7 @@ open class AgsMetrics: MetricsPublishable {
         return [
             "clientId": appData.clientId,
             "timestamp": Int(NSDate().timeIntervalSince1970 * 1000),
-            "data": payload
+            "data": payload,
```
Comment (danielpassos) [inline modules/core/metrics/AgsMetrics.swift]: Should we have `,` in the last line?

File: example/AeroGearSdkExample/auth/AuthDetailsViewController.swift [hunk 4]
```diff
@@ -76,15 +76,15 @@ class AuthDetailsViewController: UIViewController, UITableViewDataSource, UITabl
...
+
     func showServiceNotFoundDialog() {
         let alertView = UIAlertController(title: "Logout Error", message: "Auth Service is not configured. Use AgsAuth.instance.configure", preferredStyle: UIAlertControllerStyle.alert)
         alertView.addAction(UIAlertAction(title: "Ok", style: .cancel, handler: { _ in }))
         present(alertView, animated: true, completion: nil)
     }
-    
+
     func onLogoutComplete(_: Error?) {
-        self.removeView()
```
Comment (danielpassos) [inline example/AeroGearSdkExample/auth/AuthDetailsViewController.swift]: I'm not sure if we should kill `self.` dunno about the patter Apple uses, but I like it

**Pull request merged**